src/content/en/2025/accessibility.md

```diff
@@ -3,18 +3,1190 @@
+  sheets_gid="1734473098",
+  sql_file="skip_links.sql"
+  )
+}}
+
+### Document titles
+
+A descriptive page `<title>` is a basic necessity. It provides context for users navigating between browser tabs and windows and is often the first piece of information announced by a screen reader, helping users get oriented. WCAG also mandates that <a hreflang="en" href="https://www.w3.org/WAI/WCAG21/Understanding/page-titled">every page should have a meaningful title</a>.
+
+The 2025 data shows a slight improvement in the presence and descriptiveness of document titles compared to previous years. Approximately 98% of sites now include a `<title>` element, a 1% increase from 2024.
+
+This is positive. But despite this high inclusion rate, many titles remain insufficiently descriptive. This impacts usability, especially for screen reader users who rely on clear titles for orientation.
+
+There was a 2% decrease in mobile sites having titles with four or more words, which may indicate shorter or less specific titles on mobile pages. Including both a brief description of the page content and the website's name remains best practice for enhancing navigation and context.
+
+Document titles remain a fundamental accessibility feature that benefits all users. They provide context when navigating browser tabs and windows. While near-universal in presence, improving title descriptiveness and consistency continues to be an important focus in 2025 and beyond.
+
+{{ figure_markup(
+  image="title-element-statistics.png",
+  caption="Title element statistics.",
+  description="A bar chart comparing title element usage patterns on desktop and mobile. Nearly all pages (98%) have a title element on both platforms, while titles with four or more words appear on 69% of desktop pages and 67% of mobile pages. Titles that change after rendering occur on 7% of pages for both desktop and mobile.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQFD-7C6Jv6q1JyviDsKosRlVwaok7g7nRCQ9NGMw5MaAAohL7EcDejVwgp13Z_T2S_57Zi0YaVb7st/pubchart?oid=1678943411&format=interactive",
+  sheets_gid="480324977",
+  sql_file="page_title"
+  )
+}}
+
+### Tables
+
+HTML tables present data in a two-dimensional grid. Accessibility depends on structuring them with appropriate semantic elements. Using `<caption>` provides crucial context for screen reader users, while `<th>` elements define headers for rows and columns, helping users understand relationships within the data. <a hreflang="en" href="https://codepen.io/stevef/pen/ByoMebv">Steve Faulkner's tool</a>, released in 2025, can help developers quickly inspect the semantics of any HTML element.
+
+The use of `<caption>` remained steady in 2025 compared to 2024, with only a small percentage of sites including captions. This low adoption is similar to prior years: roughly 1.6% of desktop sites include captions, which is an important, though often overlooked, accessibility feature.
```

Comment:
Again can we move the chart before this line?

Can we review all charts so make sure they're inserted in the appropriate place? Since the Web Almanac is a data-driven report we really want the data to be the core that the story is told around, whereas putting the charts after the discussion almost relegates it.